deploy/service-bus/package.json

```diff
@@ -1,10 +1,9 @@
 {
   "name": "service-bus-deploy",
   "version": "1.0.0",
-  "description": "creates service bus queues",
+  "description": "manages service bus entities",
   "main": "index.js",
   "scripts": {
-    "createqs": "node ./create.js",
```

Comment:
why change the script? I think it's only an implementation change?

Reply:
`createqs` is legacy and was only left in to support the transition while it was tested in dev, before eventually implemented all the way down to production.
I am removing it because it's no longer used now.